cql3/statements/authorization_statement.cc

```diff
@@ -65,7 +65,7 @@ void cql3::statements::authorization_statement::maybe_correct_resource(auth::res
             // This is an "ALL FUNCTIONS IN KEYSPACE" resource.
             return;
         }
-        const auto& utm = qp.db().find_keyspace(*keyspace).user_types();
+        const auto utm = qp.db().find_keyspace(*keyspace).user_types();
```

Comment:
like in the `find_column_family` case, I don't like the idea of copying the user_types map, that can potentially be large.
I think this would be better:
```c++
    const auto ks = qp.db().find_keyspace(*keyspace);
    const auto& utm = ks.user_types();
```

Reply:
thank you for your review and suggestion, Benny. i incorporated your change in the latest revision.